### ISSUE TITLE
Salesforce Case 10217788

### ISSUE THREAD

**Issue by Kimberley2019** [department-of-veterans-affairs/va.gov-team#58547]
Veteran called looking to up their VA benefits but when they try to fill out the application online it tells the Veteran the VA needs more information for the Veteran to start application 21-526EZ.
When walking Veteran through the troubleshooting for this error in KM 5582 Veteran found that their profile only has "My VA, My Health, Profile, and Sign-out" options listed and no "account" feature was listed. Had Veteran check their profile to verify their personnel information and the Veteran said all their personnel information was listed and correct. Veteran states they have been trying to start this application for three days now and has been getting this same error all three days.

**Comment by Kimberley2019** [comment]
Sent case details to Joe N on Identity team for review.

**Comment by Kimberley2019** [comment]
Closing case.  Sending to EVSS for review and resolution.  See below. 
From Joe N.



Hi Kim,

I looked into the users errors and it appears as though there may be something wrong with the users data. The error we are throwing internally is "No birls_id while participant_id present". Without the participant_id the form I guess cant be started. The form itself and the rules are a little out of my realm but I would start with the IAM team and check if they can resolve the users data error. Send them the ICN and they should be able to perform a lookup.
